master/internal/checkpoint_gc.go

```diff
@@ -14,6 +14,7 @@ import (
 	"github.com/determined-ai/determined/master/pkg/tasks"
 )
 
+// TODO these are job related tasks as well.
 type checkpointGCTask struct {
```

Comment:
how do these get scheduled. what priority/weight do they take on? if I"m not mistaken they're 0 slot so they get scheduled separately